packages/plugins/rubygem-foreman_wreckingball/rubygem-foreman_wreckingball.spec

```diff
@@ -1,82 +1,51 @@
-# Generated from foreman_wreckingball-3.3.0.gem by gem2rpm -*- rpm-spec -*-
-# template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name foreman_wreckingball
-%global plugin_name wreckingball
-%global foreman_min_version 2.3
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.0
-Release: 3%{?foremandist}%{?dist}
+Name: rubygem-%{gem_name}
+Version: 5.0.0
+Release: 1%{?dist}
 Summary: Adds status checks of the VMWare VMs to Foreman
-Group: Applications/Systems
-License: GPLv3+
+License: GPLv3
 URL: https://github.com/dm-drogeriemarkt/foreman_wreckingball
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Source0: %{gem_name}-%{version}.gem
```

Comment:
This is breaking the build

Reply:
Was already at fixing it. I've used gem2rpm and the provided template. Not sure why the URL got truncated.